parsons/ngpvan/email.py

```diff
@@ -0,0 +1,143 @@
+from parsons.etl.table import Table
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Email(object):
+    def __init__(self, van_connection):
+
+        self.connection = van_connection
+
+    def get_emails(
+        self,
+        orderby=None,
+    ):
+        """
+        Get emails.
+
+        Note that it takes some time for the system to aggregate opens and click-throughs,
+        so data can be delayed up to 15 minutes.
```

Comment:
Question - is this more of an issue for `get_emails` or the iteration in `get_email_stats`?

Reply:
This language comes directly from their docs. https://docs.ngpvan.com/reference/email-overview

Reply:
I'd imagine this is more of a get_email_stats problem, since we don't pull opens and click throughs until we put in the `$expand` parameter in `/email/message/email_id`.